Automattic/newspack-newsletters
# feat: add support for Bluesky in social icons blocks

### All Submissions:






### Changes proposed in this Pull Request:



Adds support for Bluesky in social icons blocks inside the Newsletters editor. We only support a subset of available services because each one needs static PNG assets for conversion to email HTML.

### How to test the changes in this Pull Request:

1. Edit a newsletter and add a Social Icons block. 
2. Confirm that you can add a Bluesky icon/link to the block and that the icon renders with the Bluesky brand color in the editor.
3. Add another Social Icons block with a Bluesky icon/link. For this second instance, view the Styles panel and change the style to "Circle White" or "Black". (Make sure you're editing the parent Social Icons block and not the inner Bluesky block.) Confirm that the icon renders in black in the editor.
4. Send yourself a test email and confirm that both icons render in the correct color in the email.

<img width="552" alt="Screenshot 2024-12-10 at 3 55 27 PM" src="https://github.com/user-attachments/assets/2f278cec-3971-4a71-8653-99f11bea75d6">

### Other information:





---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208896118963350
